docs/pages/developer-guide.md

```diff
@@ -1550,7 +1558,7 @@ pods:
         memory: 256
         configs:
           config.xml:
-            template: "config.xml.mustache"
```

Comment:
Recently there was the change w.r.t `{{CONFIG_TEMPLATE_PATH}}` (which was not removed in `0.30.X`) is the path here consistent?

Reply:
Including it in this documentation would require also explaining that CONFIG_TEMPLATE_PATH should be manually configured in marathon.json... At this point of the doc it'd be a bit overkill.

TBH I don't expect anyone to be using this version of the dev guide anyway. Ideally they'd be against the most recent version available.